cypress.json

```diff
@@ -1,4 +1,4 @@
 {
-    "baseUrl": "http://localhost:9000/master/devhub/jordanstapinski/HEAD",
+    "baseUrl": "http://localhost:8000",
```

Comment:
This is a local setting, so I removed the prefix path from it so local development can test without having to use prefix paths (we use prefix paths for the GH Actions test)